taxi/views.py

```diff
@@ -6,7 +6,8 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 
 from .models import Driver, Car, Manufacturer
-from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
+from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm, DriverSearchForm, CarSearchForm, \
+    ManufacturerSearchForm
```

Comment:
Place all imports on a new line inside of `()`, so there would be one import per line and no imports on the same line with `(` and `)`